### ISSUE TITLE
disable scrollbar tooltip

### ISSUE THREAD

**Issue by keikoskat** [amcharts/amcharts4#904]
How do I disable the tooltip/balloon that shows up when hovering the scrollbar? 
The settings below had no effect:

```
chart.scrollbarX = new am4core.Scrollbar();

chart.scrollbarX.thumb.showSystemTooltip = false;

chart.scrollbarX.tooltip.disabled = true; 
chart.scrollbarX.hoverable = false;
chart.scrollbarX.hoverOnFocus = false,
chart.scrollbarX.tooltipText = "";
chart.scrollbarX.showSystemTooltip = false;
```

https://codepen.io/team/amcharts/pen/3826399f0a18ae210023f0a131755d50?editors=0010

**Comment by jpavansharma** [comment]
Hi,
Is there any update on this bug?
Regards,
Pavan

**Comment by martynasma** [comment]
This has been fixed in dev and will be released soon.

When released it will be enough to set `showSystemTooltip = false` on a `Scrollbar` to disable it in all of its elements:

```
chart.scrollbarX.showSystemTooltip = false;
```

**Comment by martynasma** [comment]
Fixed in [4.4.5](https://github.com/amcharts/amcharts4/releases/tag/4.4.5).

## [4.4.5] - 2019-05-13

### Changed
- Exporting chart data to JSON format now respects `dataFields` setting.

### Fixed
- Setting `showSystemTooltip = false` on a `Scrollbar` now correctly disables tooltips on its thumb and grips.
- When data for `ForceDirectedSeries` was changed for the second time, the old links remained visible.
- Initial scatter of `ForceDirectedSeries` nodes reduced. Results in a better initial animation.
- Adding data using `addData()` method to a chart with `XYChartScrollbar` was resulting data to be added twice.
- With some special combination of axes config `XYChart` sometimes could fall into infinite loop.
- Some JS error when converting coordinates out of bounds of AlbersUSA projection fixed.
- Tooltip bounds were not being copied when cloning a `Tooltip`.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
